### ISSUE TITLE
A Proper Concordance Index for Time-Varying Risk

### ISSUE THREAD

**Issue by BBolosSierra** [cmclean5/PublicHealthModels#28]
## TL;DR
This paper proposes a concordance index for time-varying risks

### Paper Link

https://doi.org/10.48550/arXiv.2208.03213

### Author/Institution

A Gandy: Department of Mathematics, Imperial College London, U.K. T Matcham: Department of Mathematics, Imperial College London, U.K. and NIHR ARC Northwest London, U.K.

## Overview

The paper suggests using the hazard rate as the time-varying risk score when calculating concordance. Through
simulations it is demonstrated situations in which other concordance indices can lead to incorrect models being selected over a true model, justifying the use of the
risk prediction in both model selection and loss functions in, e.g., deep learning models

## Methods

Survival analysis

## Comments

Chapter 6 uses DeepHit code from https://github.com/chl8856/DeepHit with changes made to also include their concordance index in the loss and validation. 

https://github.com/tmatcham/CrossingHazardConcordance
